generator/generator.js

```diff
@@ -2,67 +2,91 @@ const faker = require('faker');
+writeUsers.write('username,email\n', 'utf8');
 
-// const writeStock = fs.createWriteStream('stock.csv');
-//writeStock.write( , 'utf8');
+const writeStock = fs.createWriteStream('stock.csv');
+writeStock.write('name\n', 'utf8');
 
-// const writeBag = fs.createWriteStream('bag.csv');
-//writeBag.write( , 'utf8');
+const writeQuantity = fs.createWriteStream('quantity.csv');
+writeQuantity.write('xs,s,m,l,xl,xxl\n', 'utf8');
 
-// const writeProducts = fs.createWriteStream('products.csv');
-//writeProducts.write( , 'utf8');
+const writeStyles = fs.createWriteStream('styles.csv');
+writeStyles.write('imageUrl,styleName,price,quantityId\n', 'utf8');
 
-// const writeQuantity = fs.createWriteStream('quantity.csv');
-//writeQuantity.write( , 'utf8');
+const writeProducts = fs.createWriteStream('products.csv');
+writeProducts.write('productId,styleId\n' , 'utf8');
 
-// const writeStyles = fs.createWriteStream('styles.csv');
-//writeStyles.write( , 'utf8');
+const writeBag = fs.createWriteStream('bag.csv');
+writeBag.write('userId,itemId,size,quantity,calculatedPrice\n', 'utf8');
 
 
 const generateUser = () => {
-  let user = {
-    email: faker.internet.email(),
-    username: faker.internet.userName()
-  }
-  return user;
+
+  let email = faker.internet.email();
+  let username = faker.internet.userName();
+
+  return `${username},${email}\n`
 };
 
-// const generateProduct = () => {
-//   return faker.commerce.productName();
-// }
-
-// const generateQuantity = () => {
-//   let sizes = {
-//     xs: Math.floor(Math.random() * 10000),
-//     s: Math.floor(Math.random() * 10000),
-//     m: Math.floor(Math.random() * 10000),
-//     l: Math.floor(Math.random() * 10000),
-//     xl: Math.floor(Math.random() * 10000),
-//     xxl: Math.floor(Math.random() * 10000)
-//   }
-//   return sizes;
-// }
-
-// const generateStyles = () => {
-//   let style = {
-//     styleName: faker.commerce.color();
-//   }
-// }
-
-const writeManyEntries = (stream, amount, encoding, cb) => {
+const generateStock = () => {
+  return `${faker.commerce.productName()}\n`;
+}
+
+const generateQuantity = () => {
+  let xs = Math.floor(Math.random() * 10000);
+  let s = Math.floor(Math.random() * 10000);
+  let m = Math.floor(Math.random() * 10000);
+  let l = Math.floor(Math.random() * 10000);
+  let xl = Math.floor(Math.random() * 10000);
+  let xxl = Math.floor(Math.random() * 10000);
+
+  return `${xs},${s},${m},${l},${xl},${xxl}\n`;
+}
+
+const generateStyles = (num) => {
+
+  let imageUrl = "https://loremflickr.com/320/240/clothing";
+  let styleName = faker.commerce.color();
+  let price = faker.commerce.price();
+  let quantityId =  num;
+  return `${imageUrl},${styleName},${price},${quantityId}\n`;
+}
+
+const generateProduct = (num) => {
+  let productId = Math.floor(Math.random() * (1000000 - 1) + 1)
+  let styleId = num;
+
+  return `${productId},${styleId}\n`;
+}
+
+const generateBag = () => {
+
+  let userId = Math.floor(Math.random() * (1000000 - 1) + 1);
```

Comment:
These can be trimmed down to ` Math.floor(Math.random() * (1000000))` and achieve the same result